apps/dev-recruiters/src/components/modules/NewJoinPageComponent/RolesList/index.tsx

```diff
@@ -19,25 +19,30 @@ export function separateRoles(arr: Opportunity[]) {
     VoulunteerCordinator: [],
     SocialMediaManager: [],
   };
+
   arr.forEach((role) => {

```

Comment:
In a future PR, can we refactor this to reuse apps/dev-recruiters/src/components/modules/FilterPage/RolesFilterComponent/RolesFilterList/UseOpenPositions/opportunityUtils.tsx?

Reply:
We can try it out in future